app/models/item.rb

```diff
@@ -1,7 +1,5 @@
 class Item < ApplicationRecord
-  has_many :invoice_items
```

Comment:
why are these redundant?

Reply:
I pulled main into my branch before making the pull request, and it duplicated those lines for some reason.